lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -88,7 +88,7 @@ a.ltx_LaTeXML_logo { text-decoration: none; }
 /*======================================================================
   Para level */
 .ltx_float {
-    margin: 1ex 3em 1ex 3em; }
+    margin: clamp(1em, 4.5%, 3em) 1ex clamp(1em, 4.5%, 3em); }
```

Comment:
Did the leading `1ex` get lost, or are 3 args intentional?

Reply:
...or how to mess up a trivial CSS change... fixed!